atom/app/atom_main.cc

```diff
@@ -53,43 +96,102 @@ int ChromeMain(int argc, const char* argv[]);
 int APIENTRY wWinMain(HINSTANCE instance, HINSTANCE, wchar_t* cmd, int) {
   int argc = 0;
   wchar_t** argv_setup = ::CommandLineToArgvW(::GetCommandLineW(), &argc);
+  base::CommandLine::Init(0, nullptr);
 
-  if (IsEnvSet("ELECTRON_INTERNAL_CRASH_SERVICE")) {
-    return crash_service::Main(cmd);
-  }
+  install_static::InitializeFromPrimaryModule();
 #else  // OS_WIN
 #if defined(OS_MACOSX)
 int ChromeMain(int argc, const char* argv[]) {
 #else  // OS_MACOSX
 int main(int argc, const char* argv[]) {
 #endif
   char** argv_setup = uv_setup_args(argc, const_cast<char**>(argv));
+  base::CommandLine::Init(argc, argv_setup);
```

Comment:
Why are we calling `base::CommandLine::Init` twice?

Reply:
I'm not sure exactly, but that's what Chrome does and it seems to be necessary